clash-prelude/src/Clash/Explicit/Prelude/Safe.hs

```diff
@@ -140,6 +140,19 @@ import Clash.XException
 >>> let rP clk rst en = registerB clk rst en (8::Int,8::Int)
 -}
 
+{- $hiding
+"Clash.Explicit.Prelude.Safe" re-exports most of the Haskell "Prelude" with the
+exception of the following: @(++)@, @(!!)@, @concat@, @drop@, @even@, @foldl@,
+@foldl1@, @foldr@, @foldr1@, @head@, @init@, @iterate@, @last@, @length@,
+@map@, @odd@, @repeat@, @replicate@, @reverse@, @scanl@, @scanr@, @splitAt@,
+@tail@, @take@, @unzip@, @unzip3@, @zip@, @zip3@, @zipWith@, @zipWith3@.
+
+It instead exports the identically named functions defined in terms of
+'Vec' at "Clash.Sized.Vector". For the 'Clash.Class.Parity.odd' and
+'Clash.Class.Parity.even' function a type class called
+'Clash.Class.Parity.Parity' is available at "Clash.Class.Parity".
```

Comment:
This paragraph feels like it's missing the first sentence, I think it should be more explicit that common list functions are replaced by common vector functions.

Reply:
Changed based on discussion.